src/Passes/Source/Profiles/BaseProfile.cpp

```diff
@@ -49,16 +42,7 @@ namespace quantum
         factory.disableAliasCounting();
         factory.disableStringSupport();
 
-        function_pass_manager.addPass(TransformationRulePass(std::move(rule_set)));
-
-        // Eliminate dead code
```

Comment:
If you look at line 45 (added) then you will see that the ruleset has just been moved to a module pass instead of a function pass. All previous passes has been deprecated and replaced by a single pass "ProfilePass".

Reply:
Ah, I overlooked that.